pkg/config/config.go

```diff
@@ -374,14 +374,22 @@ type Before struct {
 
 // Blob contains config for GO CDK blob
 type Blob struct {
-	Bucket     string   `yaml:",omitempty"`
-	Provider   string   `yaml:",omitempty"`
-	Region     string   `yaml:",omitempty"`
-	DisableSSL bool     `yaml:"disableSSL,omitempty"`
-	Folder     string   `yaml:",omitempty"`
-	KMSKey     string   `yaml:",omitempty"`
-	IDs        []string `yaml:"ids,omitempty"`
-	Endpoint   string   `yaml:",omitempty"` // used for minio for example
+	Bucket     string         `yaml:",omitempty"`
+	Provider   string         `yaml:",omitempty"`
+	Region     string         `yaml:",omitempty"`
+	DisableSSL bool           `yaml:"disableSSL,omitempty"`
+	Folder     string         `yaml:",omitempty"`
+	KMSKey     string         `yaml:",omitempty"`
+	IDs        []string       `yaml:"ids,omitempty"`
+	Endpoint   string         `yaml:",omitempty"` // used for minio for example
+	Extra      BlobExtraFiles `yaml:",omitempty"`
+}
+
+// BlobExtraFiles contains config for extrafiles
+type BlobExtraFiles struct {
+	Folder string   `yaml:",omitempty"`
+	Path   string   `yaml:",omitempty"`
```

Comment:
why do we have both folder and path?

I would also suggest to use the same structure (or something similar) as we have on the release:

```go
ExtraFiles   []ExtraFile `yaml:"extra_files,omitempty"`
```

The folder inside the bucket is already provided in the `blob` section, and if the user needs extra files on different folders, they can just add more blob sections with the desired behavior...

how does that sound? 🤔

Reply:
@caarlos0 you are right, `Folder` property doesn't make sense. The `Path` property means where the files are inside the project, for example. `src/extra-fil s/*`, in this case the `yml` will looks like:

```yml
...
extra_files:
  path: src/extra-files
  files:
    - file1
```

Does this make sense to you?

Reply:
I was thinking we can use the same `ExtraFile` struct that already exists... what do you think?

Reply:
Nevermind, now I got your thought. I made the changes according to the existing ExtraFIles